cmd/kube-apiserver/app/options/options.go

```diff
@@ -113,6 +113,10 @@ func NewServerRunOptions() *ServerRunOptions {
 		},
 		ServiceNodePortRange: kubeoptions.DefaultServiceNodePortRange,
 	}
+	// here we safely assume that DefaultServiceIPCIDR is a valid CIDR value
```

Comment:
Please do not assume that

Reply:
em... why we cannot make assumptions on the *default* value?